docs/cas-server-documentation/integration/Attribute-Release-Consent.md

```diff
@@ -176,6 +192,7 @@ Endpoints must be designed to accept/process `application/json`.
 | Locate consent decision for user   | `GET`     | `principal` as headers    | `200`. The consent decisions object in the body.
 | Locate all consent decisions  | `GET`     | N/A    | `200`. The consent decisions object in the body.
 | Store consent decision    | `POST`    |  Consent decision object in the body | `200`.
+| Delete consent decision   | 'DELETE'  | `/consentDecisionId` in URL path      | `200`.
```

Comment:
`/consentDecisionId` in URL path` doesn't look right?

Reply:
Maybe so:

`/<decisionId>` appended to URL

Reply:
Looks good.